pulseguardian/management.py

```diff
@@ -9,106 +9,106 @@
 
 import requests
 
+from pulseguardian import config
 MAX_RETRY = 5
 
 
 class PulseManagementException(Exception):
     pass
 
+"""
+Wrapper around the RabbitMQ management plugin's REST API.
+"""
```

Comment:
It should be tweaked a bit too, since it's not entirely accurate for describing just the _api_request() function.